src/request/messages/ILogMessage.ts

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+export interface ILogMessage {
+    level: string;
+    message: string;
+}
```

Comment:
In the case where components expect the payloads to be a mix of message types, they can always define a composite interface that aggregates the properties of each type.  In the case where a component might need to treat the payload as completely opaque, it can use the non-generic `IMessage` type.  We'll just have to see how well that works out in practice.